fuzz.go

```diff
@@ -263,7 +273,11 @@ func (fc *fuzzerContext) doFuzz(v reflect.Value, flags uint64) {
 	defer func() { fc.curDepth-- }()
 
 	if !v.CanSet() {
-		return
+		if fc.fuzzer.allowUnexportedFields{
```

Comment:
Make sure the field is addressable -- I think it always should be but verifying seems good.

```suggestion
		if fc.fuzzer.allowUnexportedFields && v.CanAddr() {
```

Reply:
Hi Dainel!
Thanks for the suggestion! I added that.

Reply:
Hi Daniel,
Thanks for the suggestion! I added that